spec/acceptance_spec_.rb

```diff
@@ -6,94 +6,96 @@
 require 'headless'
 require 'watir'
 
-
 describe 'Acceptance Tests' do
   DatabaseHelper.setup_database_cleaner
 
   before do
-    # DatabaseHelper.wipe_database    # Kept as test data
-    # @headless = Headless.new    # need optionally install Xvfb on Mac 
-    # @headless.start
-    @browser = Watir::Browser.new
-    # @browser = Watir::Browser.new :safari
+    DatabaseHelper.wipe_database
+    @headless = Headless.new
+    @browser = Watir::Browser.new :chrome
+
+    inbox = MindMap::Entity::Inbox.new(id: nil,
+                                       name: 'Name',
+                                       url: '12345',
+                                       description: 'Description',
+                                       suggestions: [])
+
+    MindMap::Repository::Inbox::For.klass(MindMap::Entity::Inbox).find_or_create(inbox)
```

Comment:
I think the test data here is not complete yet. 
Here is only a new inbox for `/inbox/12345`, and no new suggestions.
If we want to create test data here, maybe we should also create inboxes for `/inbox` and `/inbox/guest-inbox`.

Reply:
Yep, I'm gonna finalize this tonight and then request a review